mlflow/langchain/__init__.py

```diff
@@ -271,8 +271,10 @@ def load_retriever(persist_directory):
     _validate_and_prepare_target_save_path(path)
 
     if isinstance(model_config, str):
+        if not os.path.isabs(model_config):
+            model_config = os.path.abspath(model_config)
         if os.path.exists(model_config):
-            model_config = _load_from_yaml(model_config)
+            model_config = _validate_and_get_model_config_from_file(model_config)
```

Comment:
Can we delete all the code since `_validate_and_get_model_config_from_file` is already covered here?